admin_guide/cluster_management.rst

```diff
@@ -145,3 +145,24 @@ The new shard(s) are created on the same node as the shard(s) from which the ten
     102240,
     'source_host', source_port,
     'dest_host', dest_port);
+
+Database Bloat
+##############
+
+It’s not just user queries which scale in a distributed database, vacuuming does too. In PostgreSQL big busy tables have great potential to bloat, both from lower sensitivity to PostgreSQL's vacuum scale factor, and generally because of the extent of their row churn. Splitting a table into distributed shards means both that individual shards are smaller tables and that auto-vacuum workers can parallelize over different parts of the table on different machines. Ordinarily auto-vacuum can only run one worker per table.
```

Comment:
I think this paragraph is really nice. It captures the benefits of parallelism for Vacuuming in Citus. 
I have two comments:
- We don't define what 'bloat' is. Can we briefly describe that somewhere?
- Maybe in order to do that, could we also link to PostgreSQL's Vacuum documentation? https://www.postgresql.org/docs/current/static/routine-vacuuming.html